blockchain/test_utils.go

```diff
@@ -234,22 +234,3 @@ func NewValidCloudBaseTransaction() (*Transaction, Address) {
 	}
 	return cbTx, w.Public()
 }
-
-// BigIntToHash converts a big integer to a hash
-func BigIntToHash(x *big.Int) Hash {
```

Comment:
Realized I was using this function to do some math in consensus.go, and thus moved it to hash.go.